lib/ui/screens/home_screen.dart

```diff
@@ -127,4 +128,21 @@ class _HomeScreenState extends State<HomeScreen> with LifecycleWatcher {
       ),
     );
   }
+
+  Widget customSvgIcon(String resource) {
+    return Builder(
+      builder: (BuildContext context) {
+        final iconTheme = IconTheme.of(context);
+        return SvgPicture.asset(
+          ZOStrings.iconFoodBox,
```

Comment:
Perhaps we should use the `resource` variable here?

Reply:
Omg, sure, thx 👍